addons/test_mail/models/test_mail_models.py

```diff
@@ -520,3 +520,12 @@ class MailTestPublicThread(models.Model):
     _inherit = ["mail.thread"]
 
     name = fields.Char("Name")
+
+
+class MailTestAliasRecord(models.Model):
+
+    _description = 'Fake model to test the mail alias_domain_id if add mail.alias.mixin in inherit'
+    _name = "mail.test.alias.domain"
```

Comment:
Maybe also we could add the "update" in the name, highlighting it is an upgrade issue
```suggestion
    _name = "mail.test.alias.domain.upgrade"
```